lib/core/ingestion/es_sink.rb

```diff
@@ -21,7 +21,7 @@
 module Core
   module Ingestion
     class EsSink
-      def initialize(index_name, request_pipeline, bulk_queue = Utility::BulkQueue.new, max_allowed_document_size = 5 * 1024 * 1024)
+      def initialize(index_name, request_pipeline, bulk_queue = Utility::BulkQueue.new, max_allowed_document_size = Utility::Constants::DEFAULT_MAX_INGESTION_QUEUE_BYTES)
```

Comment:
Not a big deal. Do you think we should hide `BulkQueeue` as the implementation detail of `EsSink`?

```suggestion
      def initialize(index_name, request_pipeline, max_ingestion_queue_size = Utility::Constants::DEFAULT_MAX_INGESTION_QUEUE_SIZE, max_allowed_document_size = Utility::Constants::DEFAULT_MAX_INGESTION_QUEUE_BYTES)
        @bulk_queue = Utility::BulkQueue.new(max_ingestion_queue_size, max_ingestion_queue_bytes)
```

Reply:
Yeah it's possible, maybe it's better the way you described it indeed. I also feel like moving `Utility::BulkQueue` to `Ingestion::BulkQueue` or rearranging the classes somehow better.

Reply:
A future improvement. Not necessarily now as we already passed FF.